test-requirements.txt

```diff
@@ -8,3 +8,4 @@ vcrpy
 requests_mock
 tox
 mypy
+pygobject
```

Comment:
Why is this needed?

Reply:
There is no pypi package available for libmodulemd . And actually we are only using gi package. After investigation,  It seems pygobject is good enough. So for testing, we include this package.

Reply:
I'm confused. Why is this needed only in tests? Is it because of `tox` fails to find the packages installed on the system?

Installing `python3-libmodulemd` should make Modulemd available for Python - I think it should install all necessary packages enable `from gi.repository import Modulemd`.

Reply:
Yeah. I had the same idea with you before. I thought that would be ok to install  python3-libmodulemd only. Let me resubmit it again and double check it.

Reply:
`I'm confused. Why is this needed only in tests? Is it because of tox fails to find the packages installed on the system?`
I am guessing the dependencies installed by python3-libmodulemd in system can not be used in tox environment.

Following is the failed cases with its codes:
https://github.com/redhat-exd-rebuilds/freshmaker/runs/4185263908?check_suite_focus=true
https://github.com/redhat-exd-rebuilds/freshmaker/commit/065dfdf2f1d4616d5eb054a17ecf6cead1767010

The error is: ModuleNotFoundError: No module named 'gi'